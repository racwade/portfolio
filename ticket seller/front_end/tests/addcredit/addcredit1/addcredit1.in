login
test_admin
addcredit
1001
test_buy
logout