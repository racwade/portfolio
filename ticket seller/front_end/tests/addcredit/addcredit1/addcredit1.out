welcome to the ticket seller system
enter username:
logged in as test_admin
enter event name:
enter credit amount:
enter user to add credit to:
error:invalid credit value ($1000 max)
bye